### PR TITLE
Bump github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring from 0.67.1 to 0.69.0

### PR DESCRIPTION
**Pull request by nak3**
Fixes JIRA NONE

## Proposed Changes

This patch includes https://github.com/openshift-knative/serverless-operator/pull/2371 with unit test fix.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,17 +23,17 @@ require (
 	go.uber.org/zap v1.26.0
 	golang.org/x/sync v0.4.0
 	google.golang.org/grpc v1.59.0
-	k8s.io/api v0.27.7
-	k8s.io/apimachinery v0.27.7
-	k8s.io/client-go v0.27.7
+	k8s.io/api v0.28.3
```
Comment (ReToCode) [inline go.mod:26]: Can we do that, regarding K8s min version?

Comment (nak3) [reply go.mod:26]: We use `replace` field as https://github.com/nak3/serverless-operator/blob/fix-ut/go.mod#L200 so this line is ignored.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Pull request merged**